autobuild/windows/autobuild_windowsinstaller_3_copy_files.ps1

```diff
@@ -26,20 +27,32 @@ if (($jamulus_buildversionstring -eq $null) -or ($jamulus_buildversionstring -eq
     $jamulus_buildversionstring = "NoVersion"
 }
 
+# Amend BuildOption when needed, only needed when the name build option in the artifact needs to be different
+switch ($BuildOption)
+{
+    "jackonwindows"   {$BuildOption = "jack"; break}
+}
+
 
 ###################
 ###  PROCEDURE  ###
 ###################
 
 # Rename the file
 echo "rename"
-$artifact_deploy_filename = "jamulus_${Env:jamulus_buildversionstring}_win.exe"
+if ($BuildOption -ne "")
+{
+    $artifact_deploy_filename = "jamulus_${Env:jamulus_buildversionstring}_win_${BuildOption}.exe"
+}
+else
+{
+    $artifact_deploy_filename = "jamulus_${Env:jamulus_buildversionstring}_win.exe"
+}
+
```

Comment:
If the suggestion from above is applied, this could be simplified like that?

```suggestion
$artifact_deploy_filename = "jamulus_${Env:jamulus_buildversionstring}_win${artifact_suffix}.exe"
```

Reply:
I believe that would mean you always have to add your buildoption to the switch statement, even if you do not have to change the buildoption name. I.e. an option portaudio would not need to be changed and can be used a suffix but with the above logic you would need to add this option to the switch statement to ensure the _ is added. Otherwise, if no buildoption is specified you get an _ + "" as the suffix.

Reply:
@henkdegroot, please see @hoffie's comment on lines 30-34 and note the default value of `artifact_suffix`.

Reply:
@pljones @hoffie , for me using the "artifact_suffix" is not really adding any value and will force you to always set it, even if you want to use the same as the "buildoption". Is that really what is desired? The only reason why the switch is used, is because the text "jackonwindows" is not a nice one to use in the output name of the artifact. If another option would be used, for instance "portaudio", then there is no need to change the name, but when using the artifact_suffix it is required to add this to the switch statement.

Reply:
@hoffie 
> Hm, maybe there's still a way to avoid duplicating this rather long Invoke-Native-Command call?

Not sure, I believe every option will require some kind of "if" solution. To me the current option seems to be clear when you look at the code and what it is doing.

Reply:
Agreed, see other comment thread about jackonwindows regarding the remaing points.